server/render.js

```diff
@@ -132,3 +136,36 @@ function errorToJSON (err) {
 
   return json
 }
+
+export async function serveStaticWithGzip (req, res, path) {
+  const encoding = accepts(req).encodings(['gzip'])
+  if (encoding !== 'gzip') {
+    return serveStatic(req, res, path)
+  }
+
+  const gzipPath = `${path}.gz`
+  const exists = await fs.exists(gzipPath)
```

Comment:
Seems like `fs.access` is not recommended for this: https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback

I'll work on with `renderStatic` for this to get rid of this issue.